### PR TITLE
feat: reset data menu

### PR DESCRIPTION
**Pull request by r4mmer**
### Acceptance Criteria
- Create a menu that offers the possibility to reset all wallet data for the user

### Media

> Normal call to reset

https://github.com/HathorNetwork/hathor-wallet/assets/15909384/ddf53d0f-8e00-4c81-9fc9-87cac17131c0

> Calling to reset during a bug

https://github.com/HathorNetwork/hathor-wallet/assets/15909384/ffeaffff-c88d-441d-bfaf-cea835ae4233

### CODE REVIEW

File: src/components/ModalConfirmReset.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      <div className="modal-dialog" role="document">
+        <div className="modal-content">
+          <div className="modal-header">
+            <h5 className="modal-title">
+              { t`Confirm reset all data` }
+            </h5>
+          </div>
+          <div className="modal-body">
+            <div>
+              <p>{t`Do you want to reset all of your wallet data? Only continue if you know what you are doing.`}</p>
```
[the remark this answers is not in the document]
Comment (r4mmer) [reply src/components/ModalConfirmReset.js:39]: Done

File: src/components/ModalConfirmReset.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+              <p>{t`This action cannot be undone. All your data will be lost.`}</p>
```
[the remark this answers is not in the document]
Comment (r4mmer) [reply src/components/ModalConfirmReset.js:40]: Done

File: src/components/ModalConfirmClearStorage.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            </h5>
+          </div>
+          <div className="modal-body">
+            <div>
+              <p><SpanFmt>{t`**Make sure you have the backup of your seed because this will turn your wallet into a fresh install.**`}</SpanFmt></p>
+              <p>{t`Do you want to reset all of your wallet data? Only continue if you know what you are doing.`}</p>
+              <p>{t`This action cannot be undone. All your data will be erased.`}</p>
+              <p>{t`Your wallet uniqueId will be reset.`}</p>
+              <p>uniqueId: {localStorage.getItem('app:uniqueId')}</p>
+              <p>{t`If you want to reset all data, please type '${CONFIRM_RESET_MESSAGE}' in the box below and click on 'Reset all data' button.`}</p>
```
Comment (pedroferreira1) [inline src/components/ModalConfirmClearStorage.js:46]: Does it make sense to add one more sentence like this?

```
<p>{t`After resetting the data, the application will close, and you will need to open it again.`}</p>
```

File: src/components/ModalConfirmClearStorage.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { t } from 'ttag';
+import $ from 'jquery';
+import SpanFmt from './SpanFmt';
+import { CONFIRM_RESET_MESSAGE } from '../constants';
```
Comment (pedroferreira1) [inline src/components/ModalConfirmClearStorage.js:5]: Do you have this?

File: src/components/ModalConfirmClearStorage.js (hunk 3 above)
Comment (r4mmer) [reply src/components/ModalConfirmClearStorage.js:46]: Done

File: src/components/ModalConfirmClearStorage.js (hunk 4 above)
Comment (r4mmer) [reply src/components/ModalConfirmClearStorage.js:5]: This is the same constant used by the wallet reset modal.
"I want to reset my wallet"

**Review by pedroferreira1 — APPROVED** [review]

**Review by tuliomir — APPROVED** [review]

**Pull request merged**